src/puppetlabs/kitchensink/core.clj

```diff
@@ -944,6 +944,11 @@ to be a zipper."
   []
   (str (java.util.UUID/randomUUID)))
 
+(defn uuid?
+  "Verifies whether a string is a valid UUID"
+  [uuid]
+  (re-matches #"^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$" uuid))
```

Comment:
I think it might be better to do something like this:

```clj
(try
   (UUID/fromString uuid)
   true
   (catch IllegalArgumentException
      false))
```

So that we're not re-inventing Java's UUID logic.  WDYT?